src/vendor.ts

```diff
@@ -20,6 +20,8 @@ import 'jquery';
 import 'bootstrap-loader';
 import 'font-awesome-sass-loader';
 import 'lodash';
+import 'bootstrap-datetime-picker';
```

Comment:
Could you move the import to a component file as it doesn't seem to be  a global vendor.